privacyidea/lib/auditmodules/base.py

```diff
@@ -89,8 +89,10 @@ def __init__(self, config=None):
         """
         self.name = "AuditBase"
         self.audit_data = {}
+        self.config = config or {}
         self.private = ""
         self.public = ""
+        self.have_data = False
```

Comment:
I find this irritating, that there is a object variable "have_data" and a property "has_data".

I looks like we usually could call "if f.audit_object.have_data" in before_after anyways.
I would like to have the internal variable renamed, maybe to ``self._has_data``. Thus it is more clear, that this belongs together and future audit modules could still evaluate the fact, if there is data more sophisticated in the property ``has_data``.
What do you think?

Reply:
Yeah, i was trying to be smart... I'll clean that up.

Reply:
Done in 1997585